src/components/EditPlanter.js

```diff
@@ -90,17 +90,13 @@ const EditPlanter = (props) => {
       return newPlanter[key] !== planter[key];
     });
 
-    if (changed) {
+    if (key === 'imageRotation' || changed) {
```

Comment:
This causes the _Save_ button to be enabled if there is a non-zero rotation, even if nothing has changed.